### ISSUE TITLE
Allow to add alternative that are not displayed

### ISSUE THREAD

**Issue by DegrangeM** [h5p/h5p-blanks#84]
The answer of one of my input was `0.5`,  however, as in french we use comma instead of dot, I used both solution so I set the answer to `0,5/0.5`.

However, when showing solution, student thought it was a fraction and not a list of solution and where confused.

It would be nice to allow to not display some alternative. A possibility could be to add a global option, but an other possibility could be to set for each alternative if it should be displayed as solution or not.

Maybe using double slash could be the way to do that ? So i could write `0,5//0.5` and the `0.5` solution would not be displayed.

I am able to develop this, but I wanted to know what people think of this before.

**Comment by DegrangeM** [comment]
An issue with that solution is that it doesn't allow empty solution. There is probably some teacher that are using fill in the blank in situation where the answer might be empty like : `I eat** an apple and he eat*s* an orange`.

So it need an other way to set a solution as not displayed.

Maybe `\` ? Or maybe it should be triple slash `///` ? Or maybe a character after the slash to add ?

**Comment by sr258** [comment]
I think the global option would be the most user friendly one (less markup) and it won't cause any issues with existing content, as you can easily add an upgrade script that always disables the option.

However, before jumping into the code, you should make sure that Joubel will actually accept a PR. There are already many PRs stacked up and I fear yours will suffer the same fate. You can always use your fork on your private installations, however.

**Comment by DegrangeM** [comment]
Yes after thinking the global option is probably the better. I will probably waiy for joubel opinion on this.

**Comment by otacke** [comment]
@DegrangeM Hi! I am Oliver and I was made a community maintainer for H5P Group's repositories a few weeks ago, and I am still in the process trying to tackle all the open issues and pull requests. Just to make that clear, I am not employed by H5P Group, I merely invest some of my spare time to help the H5P project.

I am terribly sorry that you have not received a reply before.

I think this is a reasonable request. I'll move the request to the status "verified by community maintainer" (see https://github.com/orgs/h5p/projects/6/views/7), meaning the pull request is to be picked up by H5P Group once it has been prioritized. Please note, however, that I don't know when they are going to handle this. My powers end here.

## Note for H5P Group

Boils down to:

- _"As an author, I want to be able to choose what correct answer option of multiple correct answer options will be shown when the users view solutions in order to reduce confusion about the correct answer and to allow internationalization."_

- _"As an author, when choosing when correct answer option should be shown, I want to be able to pick from_
  - _'Show all answer options' (current behavior) and_
  - _'Show first correct answer' (if alternative should be accepted, but the 'most correct' answer option should get focus) and_
  - _'Show correct user answer' (if alternative should be accepted, and there's no answer option that is 'more correct' than the other)."_

### Example
Blank defined as `*0,5/0.5*` Correct answer options are: `0,5` and `0.5`.
| User answer    | Behaviour setting        | Result showing up as solution                       |
| -------------: | ------------------------ | --------------------------------------------------- |
|       `0,5` ✅ | Show all answer options  | `0,5/0.5`                                           |
|       `0,5` ✅ | Show first answer option | `0,5`                                               |
|       `0,5` ✅ | Show correct user answer | `0,5`                                               |
|       `0.5` ✅ | Show all answer options  | `0,5/0.5`                                           |
|       `0.5` ✅ | Show first answer option | `0,5`                                               |
|       `0.5` ✅ | Show correct user answer | `0.5`                                               |
|         `1` ❌ | Show all answer options  | `0,5/0.5`                                           |
|         `1` ❌ | Show first answer option | `0,5`                                               |
|         `1` ❌ | Show correct user answer | `0,5/0.5` (fall back to showing all answer options) |
